samples/sceneview-background/src/main/res/layout/activity_main.xml

```diff
@@ -26,7 +26,8 @@
             android:id="@+id/transparentSceneView"
             android:layout_width="match_parent"
             android:layout_height="0dp"
-            android:layout_weight="1" />
+            android:layout_weight="1"
+            android:background="#FF0000FF"/>
```

Comment:
Yeah I added the blue color in the last version to see if the issue comes from that. I changed it back to transparent. By the way, everything ist rendering now again, maybe because of the Filament update in this version.

Reply:
Can you simply check if the `DragonAttenuation.glb` (transparency one) is displayed correctly on your devices?
It's not the case on my Pixel 4a.